ports/zephyr/v2.4/memfault_fault_handler.c

```diff
@@ -28,8 +28,8 @@
 // Since we don't use the arguments we match anything with () to avoid
 // compiler warnings and share the same bootup logic
 static int prv_install_nmi_handler() {
-  extern void z_NmiHandlerSet(void (*pHandler)(void));
-  z_NmiHandlerSet(MEMFAULT_EXC_HANDLER_NMI);
+  extern void z_arm_nmi_set_handler(void (*pHandler)(void));
```

Comment:
Is this the right file?
`ports/zephyr/v2.4/memfault_fault_handler.c`
This is for Zephyr v2.4, and back then the function was still named `z_NmiHandlerSet()`?

Reply:
Changing it there fixes the build issue. Maybe there needs to be another version.

Reply:
Right. But this might then break the build against older Zephyr versions?